.sequelizerc

```diff
@@ -0,0 +1,7 @@
+const path = require("path");
+
+module.exports = {
+  "config": path.resolve("db", "config", "config.json"),
+  "migrations-path": path.resolve("db", "migrations"),
+  "models-path": path.resolve("db", "models"),
+}
```

Comment:
Only need this because I want to keep [storing the db code in the `db/` directory while running `sequelize` command-line scripts from the project directory](http://docs.sequelizejs.com/manual/tutorial/migrations.html#the-sequelizerc-file).